submissions/Harnytskyi/friends-app/js/script.js

```diff
@@ -0,0 +1,151 @@
+const GRID = document.getElementById("allFriends");
+const CHECKBOX_MAN = document.getElementById("gender-male");
+const CHECKBOX_WOMAN = document.getElementById("gender-female");
+const SORT_AZ = document.getElementById("a-z");
+const SORT_ZA = document.getElementById("z-a");
+const SEARCH_INPUT = document.getElementById('search');
+const SORT_09 = document.getElementById("0-9");
+const SORT_90 = document.getElementById("9-0");
+let friends = [];
+
+let selectedFriends = [...friends];
+function filter() {
+    if (gender.male == true && gender.female == true)
+        selectedFriends = friends;
+    else if (gender.male == true && gender.female == false)
+        selectedFriends = friends.filter(item => item.gender == "male")
+    else if (gender.male == false && gender.female == true)
+        selectedFriends = friends.filter(item => item.gender == "female")
+}
+function displayFriends() {
+    GRID.innerHTML = '';
+    selectedFriends.forEach(item => {
+        GRID.innerHTML +=  `<div id="${item.name}" class="card">
+            <img class="photo" src="${item.photo}">
+            <p class="name">${item.name}</p>
+            <p class="age">Age: ${item.age}</p>
+            <a class="phone" href="${item.phone}">${item.phone}</a>
+            <div class="email">
+                <a href="${item.email}">
```

Comment:
Broken link. `href` argument for an email have special format

Reply:
added `mailto:`